README.md

```diff
@@ -110,3 +110,13 @@ For initial or subsequent deployment:
     ```sh
     terraform apply
     ```
+
+Note that if the public IP address changes after you set up the site initially, you will need to [change the site URL](https://codex.wordpress.org/Changing_The_Site_URL#Changing_the_Site_URL) in WordPress.
```

Comment:
I learned the hard way that WordPress saves the URL where it's accessed for the initial setup in the database, and therefore having the public IP change frequently was making parts of the site inaccessible. While a custom domain is what you'd _actually_ use, an Elastic IP is good enough for now.

Reply:
You may want to look at installing the `wp-cli` to make changes like this easier to do from the machine itself in case you can't get in to the web interface.

Reply:
Good call! Added as follow-up task: #22.